library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -212,10 +212,6 @@ $measure: 960px !default;
 
 $banner-height: 52px !default;
 
-/* Logo location */
-$logo: 'images/logo-jewel.png' !default;
-$logo-height: 36px !default;
-$logo-width: 126px !default;
```

Comment:
seemed appropriate to move to _light.scss. My goal is the majority of people shouldn't have to touch _defaults.scss to theme, just edit _light.scss or uncomment in _custom.scss to create a new theme.